cmd2/cmd2.py

```diff
@@ -211,16 +213,14 @@ def cmd_wrapper(instance, cmdline):
 
         # argparser defaults the program name to sys.argv[0]
         # we want it to be the name of our command
-        argparser.prog = func.__name__[3:]
+        argparser.prog = func.__name__[len(COMMAND_FUNC_PREFIX):]
 
         # If the description has not been set, then use the method docstring if one exists
         if argparser.description is None and func.__doc__:
             argparser.description = func.__doc__
 
-        if func.__doc__:
-            setattr(cmd_wrapper, HELP_SUMMARY, func.__doc__)
-
-        cmd_wrapper.__doc__ = argparser.format_help()
+        # If argparser.description is None, this will show under 'Undocumented commands'
+        cmd_wrapper.__doc__ = argparser.description
```

Comment:
Could possibly just default this to the previous `argparser.format_help()` value?

Reply:
That would put the usage statement back, which I think looks odd in a help summary.